pygments/lexers/procfile.py

```diff
@@ -0,0 +1,40 @@
+"""
+    pygments.lexers.procfile
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for Procfile file format.
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.lexer import RegexLexer, bygroups
+from pygments.token import Name, Number, String, Text, Punctuation
+
+__all__ = ["ProcfileLexer"]
+
+
+class ProcfileLexer(RegexLexer):
+    """
+    Lexer for Procfile file format.
+
+    The format is used to run processus on Heroku or is used by Foreman or
+    Honcho tools.
+    For more information about the definition of the format, see:
+    https://devcenter.heroku.com/articles/procfile#procfile-format
+    """
+    name = 'Procfile'
+    aliases = ['procfile']
+    filenames = ['Procfile']
+
+    tokens = {
+        'root': [
+            (r'^([a-z]+)(:)', bygroups(Name.Label, Punctuation)),
+            (r'"[^"]*"', String),
+            (r"'[^']*'", String),
+            (r'[0-9.]+', Number),
```

Comment:
This will match `..9..` as a number -- is that intentional? Could you please add a number into your Procfile example?

Reply:
I added a basic number in Procfile file and removed the dot from the regular expression so it detects `Number.Integer`. I think it's good enough because I doubt someone use float number in commands. It is ok for you?

Reply:
It's not great but I guess it'll do for now. I assume people will try to highlight valid Procfiles with it :)